crates/sui-types/src/base_types.rs

```diff
@@ -570,6 +570,14 @@ impl std::fmt::Debug for TransactionDigest {
     }
 }
 
+impl std::fmt::Debug for TransactionEffectsDigest {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::result::Result<(), std::fmt::Error> {
+        let s = base64ct::Base64::encode_string(&self.0);
```

Comment:
By readable we mean printable String? because previously it was raw (potentially unprintable) bytes?

Reply:
yeah so by default it prints out an array of bytes [54, 32, 0, 1....] which is very difficult to read :)

Reply:
Yes, this is just for parity with the TransactionDigest debug impl